tests/build.rs

```diff
@@ -5,31 +5,24 @@ pub mod fixture;
 
 use std::process::Command;
 use std::str;
-use std::sync::Mutex;
 
 use assert_cmd::prelude::*;
-use fixture::Fixture;
-
-lazy_static! {
-    static ref BUILD_LOCK: Mutex<u8> = Mutex::new(0);
-}
+use fixture::{Fixture, WranglerToml};
 
 #[test]
 fn it_builds_webpack() {
-    let fixture = Fixture::new("webpack");
+    let fixture = Fixture::new();
     fixture.scaffold_webpack();
-    fixture.create_wrangler_toml(
-        r#"
-        type = "webpack"
-    "#,
-    );
+
+    let wrangler_toml = WranglerToml::webpack_no_config("test-build-webpack");
+    fixture.create_wrangler_toml(wrangler_toml);
+
```

Comment:
creating a wrangler.toml is now as simple as picking a "template" constructor. these can be modified in place for custom tests, but the constructors that exist are helpful for the existing tests.